support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -808,8 +809,8 @@ function CheckoutComponent({
 				console.error(stripeIntentResult.error);
 			} else if (stripeIntentResult.setupIntent.payment_method) {
 				paymentFields = {
+					paymentType: Stripe,
 					stripePublicKey,
-					recaptchaToken: recaptchaToken,
```

Comment:
Where did the recaptcha token go?

Reply:
DD is the only payment method which should include this field, it was wrong before.
https://github.com/guardian/support-frontend/blob/65cc4b1a199f07eaefb9ee67f652b96864cd0341/support-frontend/app/controllers/CreateSubscriptionController.scala#L81